source/includes/stackpath/_delivery_rules.md

```diff
@@ -0,0 +1,52 @@
+## Delivery Rules
+
+Deploy and manage Delivery rules which allow you to modify StackPath CDN settings based on a number of factors like URL visited, HTTP header, and more.
+
+<!-------------------- LIST DELIVERY RULES -------------------->
```

Comment:
Left these placeholders to help during merge conflict resolution and also to maintain the order.